.prettierrc

```diff
@@ -1,4 +1,3 @@
 {
-  "singleQuote": true,
```

Comment:
Is this change relevant to this PR? Seems fine, but not sure why it's here.

Reply:
It conflicted with the stylelink rules. Could have gone either way but thought it was worth standardising.